### DISCUSSION TITLE
Using JAVA running examples CORE DOWN in debugging model with IntelliJ IDEA

### DISCUSSION THREAD

**Discussion by Wid-Mimosa** [google/or-tools#2985] [Routing (and legacy CP) questions]
Version: v8.0
Language: Java
Routing Solver
mac
When I running the demo code in this page：https://developers.google.com/optimization/routing/routing_tasks#setting-initial-routes-for-a-search，something happened cause the JRE CORE DOWN, it make me worried.
The code can be directly run and do the right thing, but when i running this code in debugging model with IntelliJ IDEA, it will core down so long as you put the breakpoint after this code,
     Assignment solution = routing.solveWithParameters(searchParameters);
and the moment of java core down happens exactly when the code running to the line where the breakpoint is added 。

I want to know what cause this problem and how to avoid this problem, because my service is online so when JVM coredown is can be terrible.

thanks for help

**Comment by EnjoyRenWei** [comment]
CBC solver?

**Comment by lperron** [comment]
Makes no sense. The routing solver does not use the cbc.
